tests/test_uflash.py

```diff
@@ -381,8 +393,8 @@ def test_extract_paths():
     mock_o.return_value.write = mock.Mock()
 
     with mock.patch('uflash.extract_script', mock_e) as mock_extract_script, \
-            mock.patch('builtins.print') as mock_print, \
-            mock.patch('builtins.open', mock_o) as mock_open:
+            mock.patch.object(builtins, 'print') as mock_print, \
+            mock.patch.object(builtins, 'open', mock_o) as mock_open:
```

Comment:
So the reason these are now *all* switched to mock.patch.object() is because it supports the 2-part form where the first argument is an object not a string.

Needed because Python 2 doesn't have a `builtins` module, but we're importing __builtin__ as builtins.